olive/passes/onnx/vitis_ai/__init__.py

```diff
@@ -2,19 +2,3 @@
 # Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
 # SPDX-License-Identifier: MIT
 #
-from onnxruntime.quantization.calibrate import CalibrationDataReader
```

Comment:
these are not passes and the vitis quantizer depends on this init file. I think it's okay to keep them here since the pass only imports then during run https://github.com/microsoft/Olive/blob/main/olive/passes/onnx/vitis_ai_quantization.py#L266

Reply:
Fixex.